src/shared/components/palette/fixed-list/read-fixed-list-field.component.ts

```diff
@@ -3,6 +3,23 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'ccd-read-fixed-list-field',
-  template: '<span class="text-16">{{caseField.value | ccdFixedList:caseField.field_type.fixed_list_items}}</span>',
+  template: '<span class="text-16">{{getValue() | ccdFixedList: getLists()}}</span>',
 })
-export class ReadFixedListFieldComponent extends AbstractFieldReadComponent {}
+export class ReadFixedListFieldComponent extends AbstractFieldReadComponent {
+
+  getValue() {
+    if (this.caseField.field_type.type === 'DynamicList') {
```

Comment:
please move this whole block of code inside `CaseField` class in an accessor method. Have a look at https://www.typescriptlang.org/docs/handbook/classes.html section 'Accessors'. Let's add a 
```
private _value: string;

    get value(): string {
        if (this.caseField.field_type.type === 'DynamicList') {
     return this._value ? this._value.value.code : '';
    } else {
      return this._value;
    }
    }
```

Reply:
Then please move the test code in read-fixed-list-field.component.spec.ts  to the one for CaseField

Reply:
Applied the code as per discussion